src/NServiceBus.AcceptanceTests/Core/Diagnostics/When_a_message_is_faulted.cs

```diff
@@ -1,11 +1,12 @@
-﻿namespace NServiceBus.AcceptanceTests.Core.Diagnostics
+﻿namespace NServiceBus.AcceptanceTests.Diagnostics
```

Comment:
```suggestion
﻿namespace NServiceBus.AcceptanceTests.Core.Diagnostics
```

looks like a mistake